api/test/e2e/route_import_test.go

```diff
@@ -577,3 +577,136 @@ func TestRoute_export_import(t *testing.T) {
 		testCaseCheck(tc, t)
 	}
 }
+
+func TestRoute_export_import_merge(t *testing.T) {
+	// create routes
+	tests := []HttpTestCase{
+		{
+			Desc:   "Create a route",
+			Object: ManagerApiExpect(t),
+			Method: http.MethodPut,
+			Path:   "/apisix/admin/routes/r1",
+			Body: `{
+					"id": "r1",
+					"uris": ["/test1", "/test2"],
+					"name": "route_all",
+					"desc": "所有",
+					"methods": ["GET"],
```

Comment:
Multiple methods should be set up to verify that the route is split after exporting, and verify the exported data is merged into one route after reimporting.

Reply:
Done